### ISSUE TITLE
Is Ai Image Generation the End or the Beginning for Graphic Artists?

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#912]
I needed an intro image for some documentation I was writing for work. 🖼️ 📝 
Searched all the standard "stock" image banks/sites and nothing good was coming up ... 🔍 ⏳ 
So I reached for the tool the "kids" are using ... 🤖 💡 
> **Note**: Not sharing the images generated for the work piece cause it's `private` (NDA, yada, yada ...)

Here's a sample image that can serve as the basis for discussion:
https://flux-ai.io/flux-ai/1830735307363778562/
<img width="1220" alt="image" src="https://github.com/user-attachments/assets/434f0be9-a93b-498a-8aad-18e5cbad9d8f">

Prompt:
```md
Create a breathtaking, intricately detailed image of a modern architectural masterpiece,
showcasing the essence of [CONSTRUCTION].

- Style: Contemporary, photorealistic
- Setting: Urban landscape, surrounded by lush greenery and bustling city life
- Lighting: Golden hour, soft and warm, casting gentle shadows
- Mood: Dynamic and inspiring, evoking a sense of innovation and progress
- Color scheme: Sleek neutrals with accents of vibrant colors,
  enhancing the structure's unique features
- Composition: Emphasize the building's striking geometry and innovative design,
  capturing its interaction with the environment and the sky's reflection on its glass surfaces

High-quality 8K render, ultra-sharp focus, rich textures.
Ensure the image conveys the building's grandeur and modernity,
highlighting its role as a centerpiece in the urban fabric.
No text or watermarks.
```

![5e3770e2afa2473eb4c1b65b320462d7](https://github.com/user-attachments/assets/307a353d-5c82-4854-9051-207d07807c6a)

This image is _incredible_. It's clearly influenced by _many_ "training" images, i.e. the prior art created by `humans`. 
But the question it raises is one worth thinking about: 

# Do AI Generated Images Spell "The End" for Graphic Artists? 

Short answer: No. It's the _beginning_. 
Longer answer: Maybe _some_ people will be put off investing the time (years!) to develop the skills to create these sorts of images. But others will be _inspired_ by the images that can be generated and motivated to learn.

The thing that struck me when _using_ the tool is that it wasn't possible to ***`edit`*** the images that were generated. 
i.e. if you want to make a minor change/tweak to something you can't!
Yes, you can tweak the **prompt** (input text) and will see a different outcome but not make _subtle_ changes. 
For that, for now, a human with graphic editing skills is required. 
And since the images generated by "Ai" are renders, there is no _source_ asset that can be edited. 
In other words, these images are not _meant_ to be edited, they are "final" renders. 

Is it only a matter of _time_ before we can highlight/circle an area of a generated image and prompt: "change this to that"?
Probably ... 💭 ⏳ 
And is the next logical step to feed the low-resolution renders into larger models and request higher resolution output? Yes.

# Personal Context: I _Love(d)_ Drawing ... 

I used to love drawing and would spend *hours* on it in a flow state. ✍️ 🎨 👨‍🎨 ❤️ 
But one day (more than a decade ago now) I spent 3 hours on a drawing ...⏳
and when I realised how much time I had spend I decided to stop. 🛑 
Not because I don't _enjoy_ it, but because it wasn't "productive" ... 💭 
i.e. I could be spending that time coding or building something that generates income. 💰 
I reasoned: "I'm never going to get paid to be a professional artist, so why invest my time in it?" 🤷‍♂️ 
Long gone was the child-like pleasure of just enjoying the task and being fully immersed in just creating ... 😍 
replace by the adult concerns of "_how will I pay this month's rent?_" 💸 

I wonder what young people will do with this technology ... 💭 
It's already being used maliciously/criminally ... https://www.bbc.com/search?q=deepfake
which, as a parent, is profoundly concerning. 😖 

But the potential for _good_ is still there. 💭

**Comment by nelsonic** [comment]
![image](https://github.com/user-attachments/assets/bc91ce58-b2cf-41ee-8f32-f97553987046)


![image](https://github.com/user-attachments/assets/4b3a2051-9359-4892-9c88-e373022fc40c)

https://flux-ai.io/flux-ai/1830644601945260033/
![image](https://github.com/user-attachments/assets/bc5f0fea-4a41-49ec-b889-b8b857ca4171)

https://flux-ai.io/flux-ai/1828100332151025666/
![image](https://github.com/user-attachments/assets/fa51823b-934e-41a3-a8b4-5024fdd8dc6b)

https://flux-ai.io/flux-ai/1826560194602250242/
![image](https://github.com/user-attachments/assets/6a6400fe-1384-4d77-b625-54982e85c265)

Try it: https://flux-ai.io/flux-ai-image-generator/

**Comment by nelsonic** [comment]
Using this example to gen an image:  https://flux-ai.io/flux-ai/1828915688986574849/
```md
Style: Vibrant anime art, slightly stylized
Character: Stephen King with a thoughtful expression and twinkling eyes
Outfit: Cozy sweater with subtle books and quills pattern, hinting at his prolific writing 
Pose: Holding a pen and an old, mystical book, exuding creativity and mystery
Background: Subtle Maine landscape elements, like forests and Victorian houses
Symbols: Include small ink pots and floating ghostly spirits
Colors: Use rich, deep colors like navy blue, maroon, and forest green
Atmosphere: Convey an intriguing mix of spooky and creative energy
Ensure the image is visually appealing and captures Stephen King's essence in a fun, anime-inspired style.
```
![image](https://github.com/user-attachments/assets/1d097dcb-2c0c-40b8-9221-c257c71b252c)

**Comment by nelsonic** [comment]
This one I'm using for "work work":

```md
Create a sticker.
Style: Vibrant anime art, cyberpunk stylized
Character: Greek Goddess Athena with a thoughtful expression, 
Outfit: smartly dressed wearing glasses, hinting prolific writing 
Pose: elegantly typing on a macbook air laptop, exuding creativity confidence, authority and reliability
Background: Subtle skyscraper city scape, sunrise
Symbols: currency symbols, banking, finance, graph
Colors: Use rich, deep colors like navy blue, gold, and forest green
Atmosphere: Convey an intriguing mix of creative energy and intelligence
Ensure the image is visually appealing and captures Greek Goddess essence in a fun, anime-inspired style.
```

![image](https://github.com/user-attachments/assets/8890897c-3213-444b-b3e8-2c492f38fa1b)

![image](https://github.com/user-attachments/assets/224bb1d0-715c-434f-a452-8461d1bb18eb)

![image](https://github.com/user-attachments/assets/987250a1-1a4a-4b51-83c9-868e61f87f70)

Low-key this is _incredible_! 🤯 

This is using the _base_ `flux.1 schnell` model which is available on HuggingFace: 
https://huggingface.co/black-forest-labs/FLUX.1-schnell
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/a9189ecb-0701-4ab9-8134-d05d661472e0">

How long before `Google` image search is _just_ AI-generated images and we lose a sense of "reality"? 💭 😬

**Comment by nelsonic** [comment]
https://medium.com/@researchgraph/the-ultimate-flux-1-hands-on-guide-067fc053fedd

**Comment by nelsonic** [comment]
https://www.reddit.com/r/StableDiffusion/comments/1f2mmcd/how_can_i_use_flux_locally/

**Comment by nelsonic** [comment]
https://deepinfra.com/blog/flux1-dev-guide

**Comment by nelsonic** [comment]
Wish I'd kept all the prompts I used with `Flux` last year ... (left them in my `Alpheya` notebook...) oh well. 
Gotta start again from scratch.

**Comment by nelsonic** [comment]
Style: Vibrant photo realistic
character: genius wizard chemist mixing magic potion liquids in a lab; no earings!
aspect ratio: 16:9 
Background: chemistry lab with modern equipment; no floating items
Symbols: elixir-lang logo, teardrop, elixir
Colors: Use rich, deep colors like navy blue, gold, and purple
Atmosphere: Convey an intriguing mix of creative energy and intelligence
Large Text at the top of the screen reads "Learn Elixir" as neon sign in exact color: #735880
Ensure the image is visually appealing and captures genius scientist essence

![learn-elixir-hero-image](https://github.com/user-attachments/assets/ed097c75-5167-427f-8a97-cfce7a0cac30)

**Comment by nelsonic** [comment]
Same prompt, same model, scientist is male: 

<img width="1375" height="1371" alt="Image" src="https://github.com/user-attachments/assets/9376bcf4-98f7-4785-bc8a-763153976b8f" />

The elixir logo on the beaker is a fail. 

So I just ran the `Nano Bana` image generated above through Flux to tidy it up: 

![Image](https://github.com/user-attachments/assets/baa1be60-5144-4acc-853f-ee37d51f470e)

This is usable now. ✅

**Comment by nelsonic** [comment]
Need an intro/hero image for the DeskAmigo Pitch. 
Through several iterations I arrived at the following prompt:

```md
Scene: A trendy Coffee Shop with stylish minimalist decor
and lots of indoor house plants make it feel like a jungle, 
pale green walls and windows letting in soft natural light. 
Show:  Side view of an elegant stylish aspirational remote working professional lady mid-20s long dark brown hair
skin has a golden tan wearing fresh trendy summer beach dress, 
with a subtle smile and bright emerald eyes sparkling with curiosity.
Details: No tattoos, jewellery, earrings or piercings just natural look. 
Sitting in a comfortable minimalist chair at a bamboo desk 
staring at a sleek modern MacBook air laptop, 
with a matcha latte on a cork coaster on the desk.
The bottom-left corner of the desk has a small sticker on it with a QR code.
the laptop screen is visible and shows an analytics dashboard 
with a graph with a positive green trend line. 
Text at the top of the screen reads “Analytics“ in exact color: #08d2ae
Environment: Coffee Shop 
The back wall of the coffee shop has a single Extra Large Neon Light Sign in Bright Lime Green #89F336 radiating light
that reads “Work Anywhere” in a quirky hand-written font.
 make the sign pop visually.
Other patrons in the coffee shop are also working in the background. Everyone else is out of focus. 
```

Used gemini (`NanoBanana Pro 3.1`) which is _free_ as part of our Google Apps:

<img alt="work anywhere intro image" src="https://github.com/user-attachments/assets/9b31d048-2ebc-4baf-86ba-bf16f13081e9" />



_Many_ failures along the way, e.g: Laptop screen on reverse side. 🙃 

<img width="2816" height="1536" alt="Image" src="https://github.com/user-attachments/assets/d4b6895b-2125-4aec-bbb3-513065df0715" />

Protagonist facing away from camera/laptop. 🤦‍♂️ 

<img width="2816" height="1536" alt="Image" src="https://github.com/user-attachments/assets/98239b31-55e6-497d-a827-2b02c11dbee6" />

**Comment by nelsonic** [comment]
Needed an illustrative image for a DevOps Engineer. 
Iterated to this prompt:

```md
Using Nano Banana 2 Pro image generation model,
Create a Photo realistic scene in 8k of a sleek minimalist modern ground-floor office at sunrise  with a simple garden outside with lawn and bamboo extends into the distance. 
natural light streaming into the building.
A physically fit male with tanned olive skin complexion senior engineer mid-30’s wearing black t-shirt with and dark jeans sits at a desk in an ergonomic Herman miller chair.  
He is wearing black Bose headphones and focussed intently on his screens.
He is staring at an array of 3 large 32-inch screens with a mechanical keyboard and ergonomic mouse. 
The text and graphics on the screens is crisp and clearly legible. 
The first screen is a status dashboard showing uptime for a cluster of servers in 3 distinct datacenter: London 99.98%, New York 99.97%, Singapore 99.95%. With the Title: “Cluster Health”
Middle Screen shows an VIM terminal with split screen with Elixir code.
Third screen shows database capacity showing 58% utilisation and other typical database monitoring metrics: NVMe disk usage, RAM usage, 12 connected nodes. 
The coffee mug on the table features a red heart logo and sits on a cork coaster. 
```

<img src="https://github.com/user-attachments/assets/9b01d181-6c84-4ece-a399-ce556ce8892d" />

Serves my purpose. ✅ 
Back to work.

**Comment by nelsonic** [comment]
Gemini occasionally _forgets_ that it has an Image Gen Model: 

<img src="https://github.com/user-attachments/assets/88ebd23a-03e9-4c7a-92dd-e4c15ebacdfc" />

So you have to _remind_ it ... 

<img  src="https://github.com/user-attachments/assets/7e4b3e17-19d5-48b6-8ba4-2d08e465db74" />

**Comment by nelsonic** [comment]
I might need a separate issue just for the bad replies I get from `Gemini Pro 3.1` ... 🤦 

Can gemini nano banana create a GIF? 

<img src="https://github.com/user-attachments/assets/742bc8da-44fe-4a51-a66b-e458622f4983" />

What is nano banana?

<img src="https://github.com/user-attachments/assets/7a06aceb-fdfe-44a4-8c8a-bc4285b58828" />

# "_Yes you're right that mushroom is poisonous ..._"

<img width="1046" height="1512" alt="Image" src="https://github.com/user-attachments/assets/c1bfb93b-e2bb-4276-9fd3-6c7cfd565560" />
